extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -621,7 +632,7 @@ export class AmpAnalytics extends AMP.BaseElement {
    */
   expandAndPostMessage_(trigger, event) {
     const msg = trigger['parentPostMessage'];
-    if (!msg || !this.isInabox_) {
+    if (!msg || !this.allowParentPostMessage_()) {
       // Only send message in inabox runtime with parentPostMessage specified.
```

Comment:
can we please update the comment as well. only send message for AMP ad with parentPostMessage specified. Thanks.

Reply:
done.